pandas_log/patched_logs_functions.py

```diff
@@ -418,34 +572,6 @@ def log__iterrows(output_df, input_df):
     return logs, tips
 
 
-def log___getitem__(output_df, input_df, key, *args, **kwargs):
```

Comment:
@cmdavis4 did you remove this method on purpose ?:)

Reply:
Yes, but I meant to fix it and reimplement it. I removed it bc it breaks if __getitem__ returns a series, eg df['some_col'], rather than df[['col_a', 'col_b']] which returns a dataframe, because the functions that check column names don't work on series. I'll try to put together a quick fix.

Reply:
Fixed